web/src/app/core/services/project.service.ts

```diff
@@ -162,15 +157,19 @@ export class ProjectService {
   }
 
   /**
-   * This function stripes the trailing slash from url
-   * @param url the url address
-   * @return the url after stripe of the trailing slash
+   * Function to update the project views
+   * @param uid uid of project
```

Comment:
We need to comment as per function arguments

Reply:
👍